source/draw/Color.ooc

```diff
@@ -178,16 +123,13 @@ ColorBgr: cover {
 	toYuva: func -> ColorYuva { ColorYuva new(this toYuv(), 255) }
 	toBgr: func -> This { this copy() }
 	toBgra: func -> ColorBgra { ColorBgra new(this copy(), 255) }
+	equals: func ~bgr (other: This) -> Bool { this blue == other blue && this green == other green && this red == other red }
 	blend: func (factor: Float, other: This) -> This {
 		This new((this blue * (1 - factor) + other blue * factor) as Byte, (this green * (1 - factor) + other green * factor) as Byte, (this red * (1 - factor) + other red * factor) as Byte)
 	}
 	distance: func (other: This) -> Float {
 		((this blue - other blue) as Float pow(2) + (this green - other green) as Float pow(2) + (this red - other red) as Float pow(2)) / 3.0f sqrt()
 	}
-	equals: func ~monochrome (other: ColorMonochrome) -> Bool { false }
-	equals: func ~yuv (other: ColorYuv) -> Bool { false }
-	equals: func ~bgr (other: This) -> Bool { this blue == other blue && this green == other green && this red == other red }
-	equals: func ~bgra (other: ColorBgra) -> Bool { false }
 	svgRGBToString: func -> String {
```

Comment:
Scalar vector graphics has nothing to do with color and I tried to move it before in a large PR that was not merged.